g11n-ws/vip-manager-l10n-agent/src/main/java/com/vmware/l10agent/utils/VipRSAUtils.java

```diff
@@ -100,13 +104,17 @@ public static String rsaEncrypt(String encryptStr, String pubKeyStr) {
 				// TODO Auto-generated catch block
 				e.printStackTrace();
 			}
-			
-			try {
-				cipher.init(Cipher.ENCRYPT_MODE, publicKey);
-			} catch (InvalidKeyException e1) {
-				// TODO Auto-generated catch block
-				e1.printStackTrace();
+			if(cipher != null) {
+			    try {
+	                cipher.init(Cipher.ENCRYPT_MODE, publicKey);
+	            } catch (InvalidKeyException e1) {
+	                // TODO Auto-generated catch block
+	                e1.printStackTrace();
```

Comment:
Please use logger

Reply:
this is a util class, and it's a history leave issue